### PR TITLE
Nojira | @ericbakenhus | Scroll to heading on event pagination change

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Scroll to top of results when paging though events

# Review By (Date)
- Now/retro

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to /events/discovery
3. Click through the pagination nav
4. Verify you are scrolled to the top of the results
5. Verify focus is moved to the results heading (results count)

### CODE REVIEW

File: src/components/events-discovery/components/Pagination/Pagination.jsx [hunk 1]
```diff
@@ -18,6 +18,31 @@ export const Pagination = (props) => {
...
+    }, 150);
+  }, []);
+
+  const handlePageSelect = useCallback(
+    (nextPage) => {
+      refine(nextPage);
+      scrollToHeading();
+    },
+    [refine, scrollToHeading]
+  );
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Pagination/Pagination.jsx:44]: Scroll and move focus to heading (`event-search-count-heading`). Wrapped in a `setTimeout` to fire after algolia does it's thing (seems to a tiny delay regardless of network time).

File: src/components/events-discovery/components/NoResults/NoResultsComponent.jsx [hunk 2]
```diff
@@ -5,7 +5,11 @@ export const NoResultsComponent = () => (
   <div className="su-w-full su-flex su-items-center su-justify-center">
     <div className="su-flex su-justify-center su-items-center su-rs-my-2">
       <div className="su-text-center">
-        <h2 className="su-text-center su-text-cardinal-red-light">
+        <h2
+          id="event-search-count-heading"
+          className="su-text-center su-text-cardinal-red-light su-scroll-mt-16"
+          tabIndex={-1}
+        >
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/NoResults/NoResultsComponent.jsx:12]: Add id, add some top scroll margin, and make programmatically focusable.

File: src/components/events-discovery/components/StatusHeader/StatusHeader.jsx [hunk 3]
```diff
@@ -53,9 +53,11 @@ export const StatusHeader = () => {
 
   return (
     <h2
+      id="event-search-count-heading"
       aria-live="polite"
       aria-atomic="true"
-      className="su-text-black-80 su-text-20 su-font-normal"
+      className="su-text-black-80 su-text-20 su-font-normal su-scroll-mt-16"
+      tabIndex={-1}
     >
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/StatusHeader/StatusHeader.jsx:61]: Same here for the "No results" heading.

Comment (ericbakenhus) [reply src/components/events-discovery/components/StatusHeader/StatusHeader.jsx:61]: Nope. Reverted. Pagination doesn't render when there are no results. Duh.

File: src/components/events-discovery/components/NoResults/NoResultsComponent.jsx (hunk 2 above)
Comment (ericbakenhus) [reply src/components/events-discovery/components/NoResults/NoResultsComponent.jsx]: Nope. Reverted. Pagination doesn't render when there are no results. Duh.

**Review by sherakama — COMMENTED** [review]
Ok

**Pull request merged**